Documentation/upgrade.md

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
oh oops, I'm not sure how this metdata/formatting line got removed.  I'll submit a quick PR to re-establish it :(

Reply:
I've opened #2225 for this